.zenodo.json

```diff
@@ -0,0 +1,28 @@
+{
+    "description": "Command line library to parse the Panda web API",
+    "license": "BSD-3-Clause",
+    "title": "dguest/pandamonium: v0.1.0",
+    "version": "v0.1.0",
+    "upload_type": "software",
+    "creators": [
+        {
+            "affiliation": "University of California Irvine",
```

Comment:
@dguest Probably want to change this to wherever you're postdocing now. I'm not sure where in Germany you are and the new CERN phonebook style has removed seeing where people are affiliated with.

Reply:
Humboldt-Universität zu Berlin, but I can't even type umlauts with the way my keyboard is set up. On other packages I just left out the affiliation and used the orcid. But yeah I should update that.